searchlib/src/vespa/searchlib/fef/indexproperties.h

```diff
@@ -347,6 +347,16 @@ namespace matching {
         static bool check(const Properties &props) { return check(props, DEFAULT_VALUE); }
         static bool check(const Properties &props, bool fallback);
     };
+
+    /**
+     * When enabled posting lists can be created on the fly even if iterator is not strict.
+     **/
+    struct CreatePostingListWithNonStrict {
```

Comment:
Consider rename to _CreatePostingListWhenNonStrict_ to match terminology elsewhere.